lib/base64.js

```diff
@@ -0,0 +1,6 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
```

Comment:
Here and elsewhere: (ESM doesn't need `use strict`):

```suggestion
```

Reply:
Fixed, thanks.